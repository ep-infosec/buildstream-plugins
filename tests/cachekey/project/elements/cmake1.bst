kind: cmake
sources:
- kind: local
  path: files/local
