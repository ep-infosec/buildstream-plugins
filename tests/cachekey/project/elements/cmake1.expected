d76a32a55035f8ca8a31634490400657f73efe6ec5d8fbc28393c98f482262de